CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # CHANGELOG
 
 ## [next]
+- Fix Box2DGame to follow render prio
```

Comment:
shouldn't be `priority` here?

Reply:
True, since that is what it is called in the code. Fixed.